patches/api/0301-Missing-Entity-Behavior-API.patch

```diff
@@ -1147,6 +1148,14 @@ index b86f0196e6eb8070830f63a94f732522c2a6c2f1..a1b42ae35dda2da90ba00a2d6666514f
 +     * @param value whether the wither can travel through portals
 +     */
 +    void setCanTravelThroughPortals(boolean value);
++
++    /**
++     * Makes the wither invulnerable for 11 seconds and
++     * sets the health to one third of the max health.
++     * <br>
++     * This is called in vanilla directly after spawning the wither.
++     */
++    void makeInvulnerable();
```

Comment:
I'm not sure about the name of this method when it also changes the current health of the wither. I get that its called that in mojang mappings, but it doesn't seem descriptive enough.

Reply:
I’m with Machine here, maybe something that links it to the game “event” like “enterInvunlerabilty“

Reply:
Maybe call it `enterInvulnerabilityPhase`?